content/reference/v1beta1/config.md

```diff
@@ -235,9 +239,23 @@ Items can be readonly.
           readonly: true
 ```
 
+When used in conjunction with a function that generates value, for example [RandomString](/reference/template-functions/static-context/#randomstring)
+- If set to `true`, the `value` is **ephemeral** between Config changes. It **cannot** be modified because it is greyed out in HTML.
+- If not set or set to `false`, the `value` is **persistent** between Config changes. It **can** be modified because its not greyed out in HTML.
 
 ### `affix`
 Items can be affixed left or right. These items will appear in the admin console on the same line.
 ```yaml
     affix: left
 ```
+
+### `help_text`
+This is similar to `description` but for `items`. This property can show a helpful message below `title`. [Markdown](https://guides.github.com/features/mastering-markdown/) syntax is supported.
+```yaml
+    - name: toggles
+      items:
+      - name: http_enabled
+        title: HTTP Enabled
+        help_text: When enabled we will listen to http
```

Comment:
kind of unrelated but I'm not totally a fan of this kind of language in a config screen, and I don't think we should encourage vendors to do this in their demo apps. Who is `we` in this case?


how about

```suggestion
        help_text: Check to enable the HTTP listener
```